src/main/java/mops/module/services/ModulService.java

```diff
@@ -0,0 +1,136 @@
+package mops.module.services;
+
+import java.lang.reflect.Field;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import lombok.RequiredArgsConstructor;
+import mops.module.database.Antrag;
+import mops.module.database.Modul;
+import mops.module.repositories.AntragRepository;
+import mops.module.repositories.ModulSnapshotRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ModulService {
+
+    private final AntragRepository antragRepository;
+    private final ModulSnapshotRepository modulSnapshotRepository;
+
+
+    /**
+     * Berechnet die Differenzen zwischen zwei Modulen.
+     *
+     * @param altesmodul Altes Modul
+     * @param neuesmodul Neues Modul
+     * @return
+     */
+    public static Modul calculateModulDiffs(Modul altesmodul, Modul neuesmodul) {
+        Modul aenderungen = new Modul();
+        boolean foundDiffs = false;
```

Comment:
Für die Einheitlichkeit "foundDiffs" vlt in Deutsch ab ändern

Reply:
Sehe ich auch so